### PR TITLE
Improve the looks of warnings, info blobs, and tips.

### PR DESCRIPTION
**Pull request by geeknoid**
Note: I didn't do the Chinese content, I'll leave that to the xlation folks.

### CODE REVIEW

File: content/about/contribute/creating-and-editing-pages/index.md [hunk 1]
```diff
@@ -460,6 +460,38 @@ will use when the user chooses to download the file. For example:
 If you don't specify the `downloadas` attribute, then the download name is taken from the `url`
 attribute instead.
 
+## Callouts
+
+You can bring special attention to blocks of content by highlighting warnings, ideas, and tips:
+
```
Comment (sdake) [inline content/about/contribute/creating-and-editing-pages/index.md]: this was a little confusing at first reading the source code, but makes sense.

File: content/blog/2018/aws-nlb/index.md [hunk 2]
```diff
@@ -16,8 +16,11 @@ Network load balancer (NLB) could be used instead of classical load balancer. Yo
 
 The following instructions require a Kubernetes **1.9.0 or newer** cluster.
 
-{{< warning_icon >}} Usage of AWS `nlb` on Kubernetes is an Alpha feature and not recommended for production clusters.
-{{< warning_icon >}} Usage of AWS `nlb` does not support the creation of two or more Kubernetes clusters running Istio in the same zone as a result of [Kubernetes Bug #69264](https://github.com/kubernetes/kubernetes/issues/69264).
+{{< warning >}}
+Usage of AWS `nlb` on Kubernetes is an Alpha feature and not recommended for production clusters.
+
```
Comment (sdake) [inline content/blog/2018/aws-nlb/index.md]: this part was updated in the blog post (by me sadly).  Should we add the tip date?

File: content/docs/setup/consul/quick-start/index.md [hunk 3]
```diff
@@ -78,13 +82,15 @@ For example, run the following command on a macOS or Linux system:
...
-> The application must use HTTP/1.1 or HTTP/2.0 protocol for all its HTTP traffic because HTTP/1.0 is not supported.
+{{< tip >}}
+Since there is no concept of pods in a Docker setup, the Istio
+sidecar runs in the same container as the application.  We will
+use [Registrator](https://gliderlabs.github.io/registrator/latest/) to
+automatically register instances of services in the Consul service
+registry.
+
+The application must use HTTP/1.1 or HTTP/2.0 protocol for all its HTTP traffic because HTTP/1.0 is not supported.
+{{< tip >}}
```
Comment (sdake) [inline content/docs/setup/consul/quick-start/index.md]: missing closing tip.

**Comment by geeknoid** [comment]
@fleeto @rootsongjc Once this goes in, kindly apply similar changes to the Chinese docs.

Thanks.

File: content/blog/2018/aws-nlb/index.md (hunk 2 above)
Comment (geeknoid) [reply content/blog/2018/aws-nlb/index.md]: Done, I added a last_update front-matter entry and a tip section about the blog having been updated.

File: content/docs/setup/consul/quick-start/index.md (hunk 3 above)
Comment (geeknoid) [reply content/docs/setup/consul/quick-start/index.md]: Fixed

**Pull request merged**